server/src/controllers/eventInstanceController.ts

```diff
@@ -9,11 +9,113 @@ import {
   validateTicketRestrictionsOnUpdate,
 } from './eventInstanceController.service';
 import {extendPrismaClient} from './PrismaClient/GetExtendedPrismaClient';
+import {parseIntToDate} from '../api/db';
 
 const prisma = extendPrismaClient();
 
 export const eventInstanceController = Router();
 
+const getDate = (time: string, date: number) => {
+  const [hour, min] = time.split('T')[1].split(':');
+  const newDate = parseIntToDate(date);
+  newDate.setHours(+hour, +min);
+  return newDate.toJSON();
+};
+
+/**
+ * @swagger
+ * /2/tickets:
```

Comment:
Should this be `/2/event-instance/tickets`? And in that case, should the summary be different?